src/foam/nanos/u2/navigation/NotificationMenuItem.js

```diff
@@ -94,9 +93,8 @@ foam.CLASS({
 
   methods: [
     function render() {
-      this.notificationDAO.on.sub(this.onDAOUpdate);
-      this.user$.dot('id').sub(this.onDAOUpdate);
-      this.group$.dot('id').sub(this.onDAOUpdate);
+      this.myNotificationDAO.on.sub(this.onDAOUpdate);
```

Comment:
Connect to detach when Element removed:
this.onDetach(this.myNotificationDAO.on.sub(this.onDAOUpdate));

Same for the next line as well.